packages/governance/src/contractGovernance/paramManager.js

```diff
@@ -44,6 +44,7 @@ const assertElectorateMatches = (paramManager, governedParams) => {
  * @property {(name: string, value: Invitation) => Promise<ParamManagerBuilder>} addInvitation
  * @property {(name: string, value: bigint) => ParamManagerBuilder} addNat
  * @property {(name: string, value: Ratio) => ParamManagerBuilder} addRatio
+ * @property {(name: string, value: Object) => ParamManagerBuilder} addRecord
```

Comment:
Nit 
```suggestion
 * @property {(name: string, value: object) => ParamManagerBuilder} addRecord
```

Though don't we have CopyRecord? I don't think it's well typed by at least it would be documentation.

Reply:
got it from endo. Seems to fit.